starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/SQLiteHelper.java

```diff
@@ -0,0 +1,96 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by jay on 10/26/16.
+ */
+
+public class SQLiteHelper {
+
+    public class SQLiteOpenHelper extends android.database.sqlite.SQLiteOpenHelper {
```

Comment:
Here you're creating an inner class inside another class. This is allowed, but I don't see any reason to do that here. Also, you have two versions of your SQLiteOpenHelper, which is confusing. I think you probably just want to delete this file altogether.